Merge final release/19.0 into trunk

AliSoftware opened the pull request:
- [x] Version bumped in `*.xcconfig` files
 - [x] `*.lproj/Localizable.strings` files updated with latest translations
 - [x] `fastlane/metadata/*` and `fastlane/jetpack_metadata/*` files updated with latest translations
 - [x] WPiOS submitted to Apple for review
 - [x] JPiOS submitted to Apple for review

Compared to the last beta, this diff adds those last-minute additions:
 - https://github.com/wordpress-mobile/WordPress-iOS/pull/17793
 - https://github.com/wordpress-mobile/WordPress-iOS/pull/17798

Commit:
Update app icon to Cool Blue

Commit:
Add blue and cool blue app icon files

Commit:
Add the old Blue icon to the Info.plist

Commit:
Change default icon to Cool Blue

Commit:
Update RELEASE-NOTES.txt

Commit:
Merge pull request #17793 from wordpress-mobile/feature/update-app-icon

Changes the default app icon to Cool Blue

mokagio committed:
Match other locales files in Jetpack `en-US` ASC metadata

This was an attempt to get the metadata uploaded to ASC. It did not
work, but it seems worth having anyway.

mokagio committed:
Add missing `description.txt` to en-US Fastlane Jetpack metadata

This is what did the trick! After adding this file to the folder, the
preview HTML that `deliver` generates also shows the en-US data 🎉

mokagio committed:
Manually update `en-US` release notes for Jetpack

GlotPress doesn't currently have an `en-US` locale to fetch, meaning the
scripts that downloads the localized metadata won't update the English
release notes. This will need to be fixed in a dedicated PR.

In the meantime, this manual change will ensure the release notes are
correct for 19.0.

mokagio committed:
Rename `name.txt` from en-US Jetpack metadata

I'm pretty confident that having it wouldn't disrupt the upload flow for
the other languages, but since I likely won't be the one running the
next upload, I want to keep the changes to a minimum.

mokagio committed:
Fix Jetpack en-US release notes not being uploaded to ASC (#17798)

AliSoftware committed:
Update translations

AliSoftware committed:
Update metadata translations

AliSoftware committed:
Bump version number

AliSoftware committed:
Update Jetpack metadata translations

mokagio commented:
Unit tests [failed on CircleCI](https://app.circleci.com/pipelines/github/wordpress-mobile/WordPress-iOS/26984/workflows/d0ad1016-2263-475d-9eb3-cd8795024988/jobs/64436?invite=true#step-106-3882) with `PostServiceWPComTests.testTrashingAPostWillUpdateItsRevisionStatusAfterSyncProperty()`. All green on Buildkite.

I restarted the build (accidentally doing "rerun workflow" instead of "rerun from failed"). I've been experiencing some flakiness on CircleCI recently. Haven't put much weight on it because we plan to decommission it soon™.

AliSoftware commented:
@mokagio the CI workflow running the tests also interrupted/cancelled the Release Build, which I had to restart, and might also explain why the Unit Test workflow were then cancelled in turn.

It happens that the Release Build (the one I relaunched after ☝️) ultimately failed anyway, because the new App Icon for AppStore got rejected for having an alpha channel (see Slack). Which means @jkmassel (who is now taking over the submission as I'm EOD) will need to update it with a new one we got without alpha, and restart the Release Build CI workflow after that.

All that meaning that the Unit Test workflow that you just restarted… might be cancelled and fail again as soon as Jeremy pushes a new commit with the asset fix, then re-triggers a Release Build (which will take priority obviously).

TL;DR: We should probably wait until after the Release Build workflow on CI finally goes green and we successfully submitted the builds to ASC before restarting the Unit Test jobs on that PR 🙃

mokagio committed:
Remove alpha channel from App Store icon

App Store Connect rejected the previous build with the following error:

```
[Transporter Error Output]: ERROR ITMS-90717: Invalid App Store Icon. The App Store Icon in the asset catalog in 'WordPress.app' can't be transparent nor contain an alpha channel.
```

@mattmiklic kindly provided a new version with no alpha, which I
verified with:

```
sips -g hasAlpha /path/to/new-icon.png
```

mokagio committed:
Bump version number to `19.0.0.5`

I did this to track the fact that we will attempt a new submission after
the `19.0.0.4` one failed because of the App Store icon including the
alpha channel, which should be fixed by
8d09f32a22f21d58b0f9cb7f135d7f25f782799d.

Also notice that I bumped the internal version to "...22", which
technically would mean the build was made on the 22nd of January, even
though (as far as UTC goes) we're still on the 21st. Pragmatically, that
has little consequence because we'll most likely won't be making any
build tomorrow.

AliSoftware commented on config/Version.internal.xcconfig:
```diff
@@ -1,9 +1,4 @@
-// and this should be the very last build for 18.3, so we should be fine.
-// – Gio
-VERSION_LONG=19.0.0.20220121
+VERSION_LONG=19.0.0.20220122
```
> Also notice that I bumped the internal version to "...22", which
technically would mean the build was made on the 22nd of January, even
though (as far as UTC goes) we're still on the 21st. Pragmatically, that
has little consequence because we'll most likely won't be making any
build tomorrow.

Can't wait for us to address https://github.com/wordpress-mobile/release-toolkit/issues/202 one day…
